src/components/button/button.tsx

```diff
@@ -20,6 +22,7 @@ export default component$(
     };
     return (
       <button
+        disabled={disabledBtn}
         class={`bg-blue-500 hover:bg-blue-700 text-white font-bold ${
           size === "regular" ? "py-2 px-4" : "py-4 px-8"
         } rounded w-auto ${colors[variant]} ${customClass}`}
```

Comment:
We should probably also change the button's color (or opacity?) when disabled

Reply:
Ok, it's not a problem